paramfetch.go

```diff
@@ -154,7 +154,8 @@ func (ft *fetch) maybeFetchAsync(ctx context.Context, name string, info paramFil
 }
 
 func (ft *fetch) checkFile(path string, info paramFile) error {
-	if os.Getenv("TRUST_PARAMS") == "1" {
+	if os.Getenv("TRUST_PARAMS") == "1" && !strings.HasSuffix(path, "vk") {
```

Comment:
I also wonder if there's a constant I should be using for this extension that doesn't only live in CGO land?

Reply:
In case of this issue we also care about the .srs file (which is also used for verification)